tests/adapters/switches/brocade_test.py

```diff
@@ -333,6 +333,31 @@ def test_get_vlan_with_a_full_interface(self):
         assert_that(str(vlan.dhcp_relay_servers[0]), equal_to('10.10.10.1'))
         assert_that(str(vlan.dhcp_relay_servers[1]), equal_to('10.10.10.2'))
 
+    def test_get_vlan_interface(self):
+        self.shell_mock.should_receive("do").with_args("show vlan 1").once().ordered().and_return(
+                vlan_display(1, 'DEFAULT-VLAN', tagged_port_str="ethe 1/2 ethe 1/23 to 1/24")
+        )
+        self.shell_mock.should_receive("do").with_args("show vlan 2").once().ordered().and_return(
+                vlan_display(1, 'DEFAULT-VLAN', untagged_port_str="ethe 1/2")
+        )
+        self.shell_mock.should_receive("do").with_args("show vlan 3").once().ordered().and_return(
+                vlan_display(1, 'DEFAULT-VLAN', untagged_port_str="ethe 1/1", tagged_port_str="ethe 1/2 ethe 1/23 to 1/24")
+        )
+
+        vlan1_interfaces, vlan2_interfaces, vlan3_interfaces = [self.switch.get_vlan_interfaces(vlan_number) for vlan_number in [1,2,3]]
```

Comment:
Something like that ?
test_get_vlan_interface_with_untagged_interface()
test_get_vlan_interface_with_tagged_interface()
test_get_vlan_interface_with_untagged_and_tagged_interface()